### PR TITLE
TOOLS-2448 `Security` feature not displayed in `features` and `summary` commands

### PR DESCRIPTION
**Pull request by jdogmcsteezy**

### CODE REVIEW

File: lib/utils/common.py [hunk 1]
```diff
@@ -235,36 +98,391 @@ def _check_value(data={}, keys=()):
...
+
+        for field_check in self._field_checks:
+            if field_check.check(sources):
+                return True
+
+        return False
+
+    def __str__(self):
+        for field_check in self._field_checks:
+            return True
```
Comment (dwelch-spike) [inline lib/utils/common.py:237]: Is this incomplete?

File: lib/utils/common.py [hunk 2]
```diff
@@ -235,36 +98,391 @@ def _check_value(data={}, keys=()):
...
+                "si_query_long_basic_complete",
+                "si_query_long_basic_error",
+                "si_query_short_basic_complete",
+                "si_query_short_basic_error",
+                "si_query_aggr_complete",
+                "si_query_aggr_error",
+                "si_query_udf_bg_complete",
+                "si_query_udf_bg_error",
+                "si_query_ops_bg_complete",
+                "si_query_ops_bg_error",
```
Comment (dwelch-spike) [inline lib/utils/common.py:342]: I don't think it will ever be the only query stats but there is also pi/si_query_abort

**Review by dwelch-spike — COMMENTED** [review]
Just a couple non-blocking comments. If they seem fine to you then this LGTM

File: lib/utils/common.py (hunk 1 above)
Comment (jdogmcsteezy) [reply lib/utils/common.py:237]: yes, good catch. Ill remove it. I was going to automate the internal docs creation but decided against it

**Review by dwelch-spike — APPROVED** [review]
LGTM

**Pull request merged**